lib/elixir/lib/list.ex

```diff
@@ -577,6 +577,51 @@ defmodule List do
   end
 
   @doc """
+  If `enum` starts with `prefix`, return true. Otherwise, return false.
+  Note that `prefix` must be a list; only `enum` is able to be an arbitrary enumerable.
+
+  ### Examples
+
+      iex> List.starts_with?([1, 2, 3], [1, 2])
+      true
+      iex> List.starts_with?([:alpha], [])
+      true
+      iex> List.starts_with?([], [:alpha])
+      false
+  """
+  @spec starts_with?(Enum.t, list) :: boolean
+  def starts_with?(enum, prefix) when is_list(enum) do
+    :lists.prefix(prefix, enum)
+  end
+  def starts_with?(_, []), do: true
```

Comment:
Preferably I wouldn't use a single line def here, since it's sandwiched between two defs that are not singlelines